x11-libs/libva-intel-media-driver/libva-intel-media-driver-21.2.2_pre.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999 ]] ; then
+	: ${EGIT_REPO_URI:="https://github.com/intel/media-driver"}
+	if [[ ${PV%9999} != "" ]] ; then
+		: ${EGIT_BRANCH:="release/${PV%.9999}"}
+	fi
+	inherit git-r3
+else
+	MY_PV="${PV%_pre}"
+	SRC_URI="https://github.com/intel/media-driver/archive/intel-media-${MY_PV}.tar.gz"
+	S="${WORKDIR}/media-driver-intel-media-${MY_PV}"
+	KEYWORDS="~amd64"
```

Comment:
Let's not add keywords for _pre releases?

Reply:
Actually any tagged version of Intel Media Driver is stable enough and is more like "RC", but officially they are named sometimes "-pre".
They are not a random snapshot of git, they are stable enough to try. FreeBSD already updated their ports.
As many packages have ebuilds for RC, beta or even alpha keyworded as "unstable", I think it makes sense to use unstable keywords for Intel Media Driver _pre releases, but never stabilize them.

Reply:
Well, I'd still prefer to not keyword release candidates.

In general, I don't really want to put things in ~arch if they're not a stable candidate.

Reply:
Keyword removed.